bluesky/plan_patterns.py

```diff
@@ -63,6 +63,99 @@ def spiral(x_motor, y_motor, x_start, y_start, x_range, y_range, dr, nth, *,
     return cyc
 
 
+def spiral_square_pattern(x_motor, y_motor, x_centre, y_centre, x_range, y_range, x_num,y_num):
+    '''Square spiral scan, centered around (x_start, y_start)
+
+    Parameters
+    ----------
+    x_motor : object, optional
+        any 'setable' object (motor, temp controller, etc.)
+    y_motor : object, optional
+        any 'setable' object (motor, temp controller, etc.)
+    x_centre : float
```

Comment:
```
x_centre : float
    x center
```
Brit-friendly naming and docstrings so that Americans can understand it, too.  💯